roles/logging/defaults/main.yml

```diff
@@ -1,4 +1,6 @@
 logging:
+  version: 0.3.1
+  logstashforwarder_url: http://repo.openstack.blueboxgrid.com/ubuntu/pool/main/l/logstash-forwarder/logstash-forwarder_0.3.1_amd64.deb
```

Comment:
this is overriden in envs to our mirror.    I'm not sure why its downloaded as a file rather than using apt from the repo, so I didn't want to mess with it.